UnknownHostException when using java -jar [jarfile]

When running a java program that uses Reflections8, with the command
```java -jar jarfile-with-dependencies.jar```
I ran into the exception listed at the bottom. This seems to be caused by ClasspathHelper just tacking `file://` before each element in the classPath and turning it into a URL. When running with `java -jar ...` this results in the URL `file://jarfile.jar` and that doesn't resolve.

I noticed that when running
```java -jar /full/path/to/the/jarfile-with-dependencies.jar```
everything works as expected, meaning we "just" need to expand the classpath element to an absolute path using `new File(entry).getAbsolutePath()`.

I will provide a pull request shortly :)

(Also, the maven.ibiblio.org site doesn't seem to work, so that tests fails)

```
Caused by: java.util.concurrent.ExecutionException: org.reflections8.ReflectionsException: Could not open url connection
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
	... 34 more
Caused by: org.reflections8.ReflectionsException: Could not open url connection
	at org.reflections8.vfs.JarInputDir$1$1.<init>(JarInputDir.java:37)
	at org.reflections8.vfs.JarInputDir$1.iterator(JarInputDir.java:33)
	at org.reflections8.Reflections.scan(Reflections.java:281)
	at org.reflections8.Reflections$1.run(Reflections.java:232)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	... 1 more
Caused by: java.net.UnknownHostException: SensorThingsProcessor-0.8-jar-with-dependencies.jar
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:220)
	at java.base/java.net.Socket.connect(Socket.java:591)
	at java.base/sun.net.ftp.impl.FtpClient.doConnect(FtpClient.java:961)
	at java.base/sun.net.ftp.impl.FtpClient.tryConnect(FtpClient.java:923)
	at java.base/sun.net.ftp.impl.FtpClient.connect(FtpClient.java:1018)
	at java.base/sun.net.ftp.impl.FtpClient.connect(FtpClient.java:1004)
	at java.base/sun.net.www.protocol.ftp.FtpURLConnection.connect(FtpURLConnection.java:295)
	at java.base/sun.net.www.protocol.ftp.FtpURLConnection.getInputStream(FtpURLConnection.java:401)
	at org.reflections8.vfs.JarInputDir$1$1.<init>(JarInputDir.java:36)
	... 8 more

```

--- a/src/main/java/org/reflections8/util/ClasspathHelper.java
+++ b/src/main/java/org/reflections8/util/ClasspathHelper.java
@@ -213,7 +213,7 @@ public abstract class ClasspathHelper {
                     .split(File.pathSeparator);
             for (String entry : classPathEntries) {
                 try {
-                    result.add(new URL("file://" + new File(entry).getAbsolutePath()));
+                    result.add(new File(entry).toURI().toURL());
                 } catch (MalformedURLException ex) {
                     if (Reflections.log.isPresent()) {
                         Reflections.log.get().warn("Could not convert "+entry+" to url.", ex);

--- a/src/main/java/org/reflections8/util/ClasspathHelper.java
+++ b/src/main/java/org/reflections8/util/ClasspathHelper.java
@@ -213,7 +213,7 @@ public abstract class ClasspathHelper {
                     .split(File.pathSeparator);
             for (String entry : classPathEntries) {
                 try {
-                    result.add(new URL("file://" + entry));
+                    result.add(new URL("file://" + new File(entry).getAbsolutePath()));
                 } catch (MalformedURLException ex) {
                     if (Reflections.log.isPresent()) {
                         Reflections.log.get().warn("Could not convert "+entry+" to url.", ex);

